origin-growth/migrations/20190124003637-create-growth-reward.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const { GrowthRewardStatuses } = require('../src/enums')
+
+const tableName = 'growth_reward'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable(tableName, {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      status: {
+        type: Sequelize.ENUM(GrowthRewardStatuses)
+      },
+      eth_address: {
+        type: Sequelize.STRING
+      },
+      campaign_id: {
+        type: Sequelize.INTEGER
+      },
+      campaign_level: {
+        type: Sequelize.INTEGER
+      },
+      campaign_rule: {
+        type: Sequelize.STRING
+      },
+      amount: {
+        type: Sequelize.INTEGER
```

Comment:
If the DB model intends to support other currencies besides OGN (like ETH for example) we should probably save values in Wei. Then `numeric` or `decimal` field types would be a more suited

Reply:
Changed to decimal.